src/prometheus_diameter_collector.erl

```diff
@@ -158,6 +161,7 @@ try_dict(Dict, {_, CmdCode, Rbit} = Cmd) ->
 	    Cmd
     end.
 
+msg_name({_, _}, _Apps) -> unknown;
```

Comment:
Not sure but maybe will be safe if add this matching as last function clause: `msg_name(_, _Apps) -> unknown;`
And add log information about it.

Reply:
good idea, changed. 
I don't really want to log in this library/module. If/when this happens, it can be observed in the metrics.